core/src/main/java/com/linecorp/armeria/client/ClientRequestContextBuilder.java

```diff
@@ -91,6 +118,12 @@ public ClientRequestContextBuilder connectionTimings(ClientConnectionTimings con
         return this;
     }
 
+    @VisibleForTesting
+    ClientRequestContextBuilder enableTimeout(boolean enableTimeout) {
```

Comment:
What's the difference between `enableTimeout` and `timedOut`?
Can't we combine them?

Reply:
The difference between them is:
* `enableTimeout(false)` does not set `DefaultTimeoutController` when building `ClientRequestContext`. This is useful when testing pending timeout task because we can set it later via `DefaultClientRequestContext.setResponseTimeoutController()`. TimeoutController can be set only once.
https://github.com/line/armeria/blob/a7318e458e5306babd2e5e259bdf27d740617c1b/core/src/main/java/com/linecorp/armeria/internal/common/TimeoutScheduler.java#L184-L187
* As you know, just inject a TimeoutController that has been timed-out.

Reply:
Thanks for the explanation. I felt that `enableTimeout(false)` meant that the timeout is disabled for this context so we cannot schedule it.
How about renaming something like `configureDefaultTimeoutController`?

Reply:
> I felt that enableTimeout(false) meant that the timeout is disabled for this context so we cannot schedule it.

It seems better to remove the boolean argument. How about renaming to `noTimeoutController()`?

Reply:
Sounds good to me. 👍